recipe/meta.yaml

```diff
@@ -35,40 +35,47 @@ requirements:
     - click-plugins
     - cligj >=0.5
     - libgdal
-    - setuptools
-    - snuggs >=1.4.1
     - importlib-metadata  # [py<310]
-    - {{ pin_compatible('numpy') }}
+    - numpy >=1.24
```

Comment:
you're getting an overdepending error concerning numpy, any idea why?

Reply:
Hmm.. I'm not sure. It definitely seems to be used. https://github.com/search?q=repo%3Arasterio%2Frasterio%20numpy&type=code